### ISSUE TITLE
PHP 8.1 images are only installing PHP 8.2

### ISSUE THREAD

**Issue by dragonfire1119** [serversideup/docker-php#101]
### Affected Docker Images

serversideup/php:8.1-fpm-nginx

### Current Behavior

I've had this Dockerfile building on CircleCI for a while without changing anything, and now it's building with composer complaining about extensions not being found. I'm also noticing that PHP 8.2 extensions are being loaded instead of 8.1, which could explain why.

php8.1-redis installs php8.2-redis.

### Expected Behavior

It should be installing 8.1 php extensions.

### Steps To Reproduce

```
RUN apt-get update && apt-get install -y \
    php8.1-redis \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*
```

You should see it not installing 8.1 and only 8.2 in the console.

### Host Operating System

Circle CI

### Docker Version

```markdown
Client:
 Cloud integration: v1.0.29
 Version:           20.10.21
 API version:       1.41
 Go version:        go1.18.7
 Git commit:        baeda1f
 Built:             Tue Oct 25 18:01:18 2022
 OS/Arch:           darwin/arm64
 Context:           desktop-linux
 Experimental:      true

Server: Docker Desktop 4.14.0 (91374)
 Engine:
  Version:          20.10.21
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.7
  Git commit:       3056208
  Built:            Tue Oct 25 17:59:41 2022
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.9
  GitCommit:        1c90a442489720eec95342e1789ee8a5e1b9536f
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### Anything else?

Seems like it's only installing 8.2 in 8.1 images.

**Comment by jaydrogers** [comment]
I see you closed the issue by the time I got to it. I hope that means you found a solution 😃

I just did a quick look and confirmed that it seems to be acting normally. If you're still not seeing the correct versions, let me know!

**Comment by dragonfire1119** [comment]
> I see you closed the issue by the time I got to it. I hope that means you found a solution 😃
> 
> I just did a quick look and confirmed that it seems to be acting normally. If you're still not seeing the correct versions, let me know!

Thank you for responding, but I did resolve this. My ARG for PHP VERSION was empty because it was not under the from, and now that 8.2 is available, Ubuntu apt was defaulting to 8.2 instead of 8.1.

**Comment by dragonfire1119** [comment]
> I see you closed the issue by the time I got to it. I hope that means you found a solution 😃
> 
> I just did a quick look and confirmed that it seems to be acting normally. If you're still not seeing the correct versions, let me know!

Thank you for responding, but I did resolve this. My docker ARG for PHP_VERSION was empty because it was not under the from, and now that 8.2 is available, Ubuntu apt was defaulting to 8.2 instead of 8.1.

**Comment by jaydrogers** [comment]
Thanks for sharing your detailed solution 🙌
